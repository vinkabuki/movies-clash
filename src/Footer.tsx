import React from 'react'
import styled from 'styled-components'

const FooterText = styled.p`
  background-color: rgba(0, 0, 0, 0.1);
`

const Footer: React.FC = () => {
  return <FooterText>Thank you for using Movies Clash</FooterText>
}

export default Footer
