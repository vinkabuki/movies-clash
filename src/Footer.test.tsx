import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

describe('Footer component basic test', () => {
  test('Render footer component', () => {
    const component = renderer.create(<Footer />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
