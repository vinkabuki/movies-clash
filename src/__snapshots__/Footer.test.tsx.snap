// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component basic test Render footer component 1`] = `
<p
  className="sc-AxjAm gvWQIK"
>
  Thank you for using Movies Clash
</p>
`;
